localstack/services/cloudwatch/alarm_scheduler.py

```diff
@@ -276,37 +284,58 @@ def calculate_alarm_state(alarm_arn: str) -> None:
     alarm_details = get_metric_alarm_details_for_alarm_arn(alarm_arn)
     client = get_cloudwatch_client_for_region_of_alarm(alarm_arn)
 
+    query_date = datetime.utcnow().strftime(format="%Y-%m-%dT%H:%M:%S+0000")
     metric_values = collect_metric_data(alarm_details, client)
 
+    state_reason_data = {
+        "version": "1.0",
+        "queryDate": query_date,
+        "period": alarm_details["Period"],
+        "recentDatapoints": [v for v in metric_values if v],
```

Comment:
This would remove values==0. Is this intended?

Reply:
no, not intended - thanks for pointing this out 🙏